nextflow.config

```diff
@@ -69,6 +69,10 @@ params {
     schema_ignore_params  = null
 }
 
+// Container slug. Stable releases should specify release tag!
+// Developmental code should specify :dev
+process.container = 'qbicpipelines/rnadeseq:dev'
+
```

Comment:
```suggestion
```

Reply:
Is there a reason why this was added again? We're in DSL2 so the container is specified within the module, or?

Reply:
Oop, you're right of course!